miniApp/smartNotes/views.py

```diff
@@ -1,3 +1,5 @@
 from django.shortcuts import render
-
+from django.http import HttpResponse
 # Create your views here.
+def index(request):
+    pass
```

Comment:
put an extra line on bottom.

Reply:
Means to put an extra line before 'pass' or after 'pass'?

Reply:
Resolved it